packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -56,7 +56,7 @@ export default function ShareOptionsMenu({
   onHidden,
   contextId = 'share-context',
 }: ShareOptionsMenuProps): ReactElement {
-  const link = post && post?.permalink;
```

Comment:
Why my previous commit here? It has already been merged to the main. And what can I do to avoid it? + @rebelchris @idoshamun

Reply:
Not sure actually. Maybe you merged the updates into your feature branch and this caused this problem

Reply:
Probably if you repull from main it will change this :)

Reply:
this is the same previous branch. We used to use the same branch for small fixes. I guess I need to use a new branch for each pr because of squash?

Reply:
Ah yes, we normally repull main -> then create a new individual branch for hotfixes.